### PR TITLE
feat: creat test.txt

### PR DESCRIPTION
**Pull request by Huy1711**
Should be generated from a PR template

Authors:
- @Huy1711 
- @cin-kay 

Descriptions:
...

### CODE REVIEW

**Review by hungpham3112 — APPROVED** [review]
LGTM

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello world
```
Comment (khankaholic) [inline test.txt:1]: @Huy1711 please fix this, this should be `hi` not `hello`

**Review by khankaholic — APPROVED** [review]
LGTM

**Comment by khankaholic** [comment]
Close due to testing rules

**Pull request closed without merging**